pycaption/exceptions.py

```diff
@@ -24,4 +24,8 @@ class RelativizationError(Exception):
     """
     Error raised when absolute positioning cannot be converted to
     percentage
-    """
\ No newline at end of file
+    """
+
+class InvalidInputError(Exception):
```

Comment:
Client code might have caught RuntimeError in situations where now this is raised. We should try our best to avoid backwards incompatibilities, and I think we can do it in this situation.